util/archutil/loong64_check_loong64.go

```diff
@@ -0,0 +1,8 @@
+//go:build loong64
+// +build loong64
```

Comment:
Probably these Ines are not needed

Reply:
solved

Reply:
Other archs seem to be consistent, I'm not sure if it needs to be retained